packages/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -16,15 +16,13 @@ URL: https://%{goimport}
 Source0: https://%{goimport}/archive/v%{gover}/nvidia-container-toolkit-%{gover}.tar.gz
 Source1: nvidia-container-toolkit-config-k8s.toml
 Source2: nvidia-container-toolkit-config-ecs.toml
-Source3: nvidia-oci-hooks-json
```

Comment:
I asked @sky1122 to keep this change and the removal of `nvidia-oci-hooks-json` in this commit and not in the other commit that modifies `nvidia-container-toolkit` because removing this file was used by `shimpei`.